dcos/cluster.py

```diff
@@ -182,8 +182,13 @@ def get_cluster_cert(dcos_url):
         with urlopen(cert_bundle_url, context=unverified) as f:
             return f.read().decode('utf-8')
     except Exception as e:
-        msg = "Error downloading CA cert from cluster"
-        raise DCOSException("{}:\n{}".format(msg, e))
+        # DC/OS open clusters do not have certs
+        if e.code == 404:
```

Comment:
if the exception here does not inherit from requests' HTTPError then `e` may not have the `code` property, resulting in an AttributeError. A TCP layer problem might cause this (ConnectionRefusedError etc).